config/locales/en.yml

```diff
@@ -437,6 +437,12 @@ en:
     creation_type:
       manual: 'manually created'
       background: 'automatically created'
+    reasons_cannot_charge:
```

Comment:
@guicassolato I think the names here are kind of counter-intuitive. It makes me think that `not_paid`, `positive`, and `provider_present` are reasons why the invoice **cannot** be charged, which is quite opposite.

And well, `buyer_account_paying_monthly: 'buyer not paying monthly'` itself is a bit confusing 😅

Reply:
TBH, I struggled with that too yesterday. In the end, I kept it that way because it makes the whole code to evaluate the conditions and fetch corresponding error messages cleaner.

Thanks for raising it anyways. I'll try to improve that in the next iteration.